docs/workflow/testing/libraries/testing-wasm.md

```diff
@@ -100,12 +100,15 @@ The following shows how to run tests for a specific library
 
 - `$(WasmXHarnessArgs)` - xharness command arguments
 
-    Example: `WasmXHarnessArgs="--xyz"` -> becomes `dotnet xharness wasm test --xyz`
+    Example: `WasmXHarnessArgs="--set-web-server-http-env=DOTNET_TEST_WEBSOCKETHOST"` -> becomes `dotnet xharness wasm test --set-web-server-http-env=DOTNET_TEST_WEBSOCKETHOST`
 
 - `$(WasmXHarnessMonoArgs)` - arguments to mono
 
     Example: `WasmXHarnessMonoArgs="--runtime-arg=--trace=E"`
 
+- `$(WasmXHarnessMonoArgs)` - env variables to mono
```

Comment:
Should this be folded into L105 because it is still the same variable? Perhaps `arguments/env variables to mono` with both examples.

Reply:
fixed